buildkite/common

```diff
@@ -29,15 +29,14 @@ export DB_PORT=5432
 export NETWORK=test_nw
 export GIT_SHA=$GIT_SHA
 
-# Temporary measure to install constellation on all build steps - required by orderly-web install
-pip3 install constellation
-
 # Teardown on exit
 function cleanup() {
-  $HERE/../scripts/stop-database.sh
-  $HERE/../scripts/stop-api.sh
-  $HERE/../scripts/stop-orderly-web.sh
-  $HERE/../scripts/stop-task-queue.sh
+  $HERE/../scripts/stop-database.sh || true
```

Comment:
usually we do `set +e` here to disable errors and then you don't need the `|| true` bit